python/UniverseWrapper.cpp

```diff
@@ -272,6 +273,42 @@ namespace FreeOrionPython {
             .def(py::map_indexing_suite<Effect::AccountingMap, true>())
         ;
 
+        //////////////////////////////////////////////
+        //   Content Named Global Constant Values   //
+        //////////////////////////////////////////////
+        py::def("namedRealDefined",
+                +[](const std::string& name) -> bool { return GetValueRef<double>(name, true); },
+                "Returns true/false (boolean) whether there is a defined double-valued "
+                "scripted constant with name (string).");
+        py::def("getNamedReal",
+                +[](const std::string& name) -> double {
+                    auto ref = GetValueRef<double>(name, true);
+                    if (!ref)
+                        return 0.0;
```

Comment:
Can we please not return 0.0 here and in the int case below?

This will inevitably introduce bugs where FOCS constants are renamed but some AI code will just silently use zero in calculations instead of throwing an error (which could be rather impactful like silently multiplying with zero).

I'd rather return ```None``` and have the python code throw if an outdated constant is used in a math context.

IIRC, it should be possible by returning a ```boost::python::object``` instead of a double/int - specifically ```return py::object()``` on failure and wrapping the int/double return value like ```return py::object(retval)``` on success.

Reply:
hopefully OK now